packages/databricks-vscode/src/test/e2e/wdio.conf.ts

```diff
@@ -117,7 +117,7 @@ export const config: Options.Testrunner = {
         return [
             {
                 "browserName": "vscode",
-                "browserVersion": engines.vscode.replace("^", ""),
+                "browserVersion": "stable",
```

Comment:
Due to chromedriver issues we can't actually test the minimal supported version while using the `6.0.0` version of `wdio-vscode-service`. We can work around `5.*` issues and don't update, but I'm not sure it's worth it. 

Testing on stable makes sense since that's what our users use. But I agree that it's useful to test the minimal supported version. I'll add a test-matrix on the github actions level for that in a separate PR, but I think I'll just use the minimal vscode version that works with the new wdio service (1.86, instead of our 1.83)

Reply:
Let's bump our minimum supported version to 1.86 as well. We are anyway pitching this as the v2. Might as well take it to the highest version.

Reply:
Yeah, that works too, although it still makes sense to run test under the moving "stable" target.